README.md

```diff
@@ -150,7 +150,36 @@ Custom translations of properties can be added by extending the
 
 (defn -main [& args]
   (jdbc/with-db-connection [conn {:datasource datasource}]
-    (let [rows (jdbc/query conn "SELECT * FROM table")]
+    (let [rows (jdbc/query conn "SELECT 0")]
```

Comment:
select 0 is a simple working query sybase and postgres.  The equivalent in Oracle is select 0 from dual.